cfgov/unprocessed/js/modules/transition/MoveTransition.js

```diff
@@ -0,0 +1,160 @@
+'use strict';
+
+// Required modules.
+var EventObserver = require( '../../modules/util/EventObserver' );
+var BaseTransition = require( './BaseTransition' );
+
+// Exported constants.
+var CLASSES = Object.seal( {
```

Comment:
Using `Object.seal` won't prevent someone from changing the existing values. it will prevent someone from adding new values though.